portfolio/src/main/webapp/index.html

```diff
@@ -50,6 +50,15 @@ <h1>My Portfolio</h1>
         <button onclick="window.location.href='books.html';"/>Books</button>
         <button onclick="window.location.href='projects.html';"/>My Projects</button>
         <button onclick="getRandomQuoteUsingArrowFunctions()"/>Fun Fact!</button>
+        <div id='fun-fact'></div>
+        <form action="/comments" method="POST">
+          <p>Leave a comment:</p>
+          <textarea name="comment-input"></textarea>
+          <br/><br/>
```

Comment:
Don't close `<br>` elements or other 'void' elements (i.e., ones with no inner html). ([Style guide link](https://google.github.io/styleguide/htmlcssguide.html#Document_Type))
```suggestion
          <br><br>
```